app/experimenter/urls.py

```diff
@@ -9,6 +9,9 @@
     re_path(r"^api/v1/experiments/", include("experimenter.experiments.api.v1.urls")),
     re_path(r"^api/v2/experiments/", include("experimenter.experiments.api.v2.urls")),
     re_path(r"^api/v3/", include("experimenter.experiments.api.v3.urls")),
+    re_path(
+        r"^api/v3/visualizations/", include("experimenter.visualization.api.v1.urls")
```

Comment:
Ah good thank you for reordering that, but I also think we could:

1) Exclude the `visualizations/` subpath here so it looks the same as the regex matcher above it (Django will merge these two together properly
2) It's confusing to put 'v3' in the url but 'v1' in the import path, lumping it together with v3 seems reasonable to me since that's already waht' sused as the private API for the frontend, but maybe name the folder `visualization/api/v3` so that it matches its public interface?